src/sleep.cc

```diff
@@ -41,7 +41,7 @@ void SleepForMicroseconds(int microseconds) {
 }
 
 void SleepForMilliseconds(int milliseconds) {
-  SleepForMicroseconds(static_cast<int>(milliseconds) * kNumMicrosPerMilli);
+  SleepForMicroseconds(static_cast<int>(milliseconds * kNumMicrosPerMilli));
```

Comment:
Otherwise ICC produces: non-pointer conversion from "long" to "int" may lose significant bits. I could suppress the warning. But I assumed that a cast would be better. Also it seemed that the static_cast as it was made no sense. Given that milliseconds is already int.